### PR TITLE
python-launcher 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by brettcannon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/python-launcher.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class PythonLauncher < Formula
+  desc "Launch your Python interpreter the lazy/smart way"
+  homepage "https://github.com/brettcannon/python-launcher"
+  url "https://github.com/brettcannon/python-launcher/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "31e5a4e50e3db4506e8484db06f6503df1225f482b40a892ffb5131b4ec11a43"
+  license "MIT"
+
```
Comment (carlocab) [inline Formula/python-launcher.rb:7]:
```suggestion
```

File: Formula/python-launcher.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  head "https://github.com/brettcannon/python-launcher.git", branch: "main"
+
+  livecheck do
+    url :head
+    regex(/^v(\d+\.\d+\.\d+)$/i)
+  end
```
Comment (carlocab) [inline Formula/python-launcher.rb:13]: Do we need this?

**Review by carlocab — COMMENTED** [review]
Thanks, @brettcannon. Just a few comments. Please squash further changes into your first commit.

File: Formula/python-launcher.rb (hunk 2 above)
Comment (nandahkrishna) [reply Formula/python-launcher.rb:13]: We can remove this, there aren't any non-standard tag names in the repository.

**Comment by brettcannon** [comment]
@carlocab @nandahkrishna   comments resolved and everything has been squashed. PTAL.

**Comment by brettcannon** [comment]
BTW I'm assuming there's nothing else needed for this to be made available for both macOS and Linux? I didn't see anything specific in the formula cookbook about opting in or out specifically to both platforms, so I assume what I have is fine?

**Review by carlocab — APPROVED** [review]
Thanks, @brettcannon.

For future reference, our preferred commit message format here is the same as the title:
```
python-launcher 1.0.0 (new formula)
```
The format for version bump PRs is identical -- just leave out the `(new formula)` part.

**Comment by brettcannon** [comment]
Thanks! I'll try to remember about the commit message for any future version bump PRs.

**Comment by brettcannon** [comment]
Unfortunately there appears to have been an issue in getting this formula into Linuxbrew. I have opened https://github.com/Homebrew/linuxbrew-core/issues/23953 to track the problem (which clang-format is also having in https://github.com/Homebrew/linuxbrew-core/issues/23906).

**Pull request closed without merging**